domains/thenamelessdev.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "thenamelessdev",
+    "discord": "thenamelessdev",
+    "email": "aron@thenamelessdev.com"
+  },
+  "records": {
+    "CNAME": "thenamelessdev.github.io",
+    "TXT": "43d7a8027143f53857f30091b321cc"
```

this is not how this works
```suggestion
    "CNAME": "thenamelessdev.github.io"
```